docs/2-developing/0-connectors/2-connector-specification.mdx

```diff
@@ -2,25 +2,156 @@
 title: "Connector Specification"
 ---
 
-[Specification](https://pkg.go.dev/github.com/conduitio/conduit-connector-sdk#Specification) contains general information regarding the plugin like its name and what it does.
+A connector's specification is used to present information about the connector
+and validate connector parameters. The specification can be found in the
+`connectors.yaml` file, that's found in the root of a connector's repository.
+The following is an example of
+the [file connector's](https://github.com/ConduitIO/conduit-connector-file)
+`connector.yaml`:
 
-### `spec.go`
+```yaml
+version: "1.0"
+specification:
+  name: file
+  summary: A file source and destination plugin for Conduit.
+  description: |-
+    The file source allows you to listen to a local file and
+    detect any changes happening to it. Each change will create a new record. The
+    destination allows you to write record payloads to a destination file, each new
+    record payload is appended to the file in a new line.
+  version: v0.10.0
+  author: Meroxa, Inc.
+  source:
+    parameters:
+      - name: path
+        description: Path is the file path used by the connector to read/write records.
+        type: string
+        default: ""
+        validations:
+          - type: required
+            value: ""
+    # other parameters
+```
+
+The contents of this file are embedded into a connector's binary and read by the
```

Comment:
I'm guessing the developer is the one who should embed it right? like in the example that has `//go:embed`, we should let the developer know that this is needed, and won't just happen on its own.
I might've misunderstood something here, but if the line `//go:embed connector.yaml` is not added, then the file won't be embedded right?